askup/templates/askup/qset_mixed.html

```diff
@@ -1,47 +1,32 @@
 {% extends 'askup/base.html' %}
 
 {% block body %}
-    <div class="container-fluid">
-        <div class="row">
-            <h3 class="center">
-                    {{main_title}}
-                    <a class="btn shortcut-button-link" id="edit-qset" data-toggle="modal" data-target="#modal-edit-qset">
-                        <span class="glyphicon glyphicon-edit shortcut-button-glyphicon"></span>
-                    </a>
-             <!-- todo: delete button here? -->
-            </h3>
-        </div>
-        <div class="row">
-            <div class="col-xs-12">
-                <h5 class="center">
-                    <span id="breadcrumbs" class="center">
+<div class="container-fluid" id="qset-show-container" data-qset-id="{{ current_qset_id }}">
+    {% include "askup/qset_top_controls_large.html" %}
```

Comment:
why do we need two html files? small and large they look like copy past one from other?